wallet.go

```diff
@@ -1262,6 +1420,37 @@ func (w *Wallet) TotalReceived(confirms int) (btcutil.Amount, error) {
 	return amount, nil
 }
 
+// TotalReceivedForAccount iterates through a wallet's transaction history,
+// returning the total amount of bitcoins received for a single wallet
+// account.
+func (w *Wallet) TotalReceivedForAccount(account uint32, confirms int) (btcutil.Amount, uint64, error) {
```

Comment:
Right, we need `TotalReceivedForAccount`, but `TotalReceived` is no longer used and is dead code.